components/molecule/coachmark/src/Tooltip/index.js

```diff
@@ -37,15 +37,8 @@ const Tooltip = ({
         ) : null}
 
         {tooltipOptions.closeIcon ? (
-          <div className={`${BASE_CLASS}-tooltipCloseButton`}>
-            <AtomButton
```

Comment:
🍆 Is the removal of the AtomButton something expected?

Reply:
From UX they decide to remove it and use whatever icon is passed as option.